webapp/src/modules/game/game.redux.js

```diff
@@ -18,15 +18,22 @@ export const { Types: GameTypes, Creators: GameActions } = createActions({
   pokeAnimal: ['fieldIndex'],
   buyAnimal: ['fieldIndex', 'cost'],
   upgradeWarehouse: ['fieldIndex', 'cost'],
+  hideInstruction: null,
```

Comment:
Personally, I feel that `userAuth` module would be better for holding information regarding showing or not an instruction for the user, but I do not insist.

Reply:
I'd rather leave it here to save time, it is kind of game state after all